jobs/kpi-forecasting/kpi_forecasting/configs/search_forecasting_model_performance.yaml

```diff
@@ -0,0 +1,6 @@
+input_config_list: ["search_forecasting_ad_clicks.yaml",  "search_forecasting_daily_active_users.yaml", "search_forecasting_search_count.yaml"]
+output_project: moz-fx-data-bq-data-science  # change before merging
+output_dataset: jsnyder # change before merging
+output_table: "search_validation" # change before merging
```

Comment:
Before merging, let's change the table name to include "forecast", e.g. "search_forecast_validation"